src/app/public/modules/autocomplete/autocomplete.component.html

```diff
@@ -26,6 +26,11 @@
           </button>
 
         </sky-dropdown-item>
+        <sky-dropdown-item *ngIf="searchResults.length === 0">
+          <div class="sky-padding-squish-default">
```

Comment:
the "not found" text should use sky-deemphasized class

Reply:
👍 Updated!